datapreview.tex

```diff
@@ -60,28 +60,107 @@ \subsubsection{Data Preview 0.3}
+DP1 is expected to include of the order of 2000 science images.
+\input{tables/dp1_fields_visits}
+
+\begin{figure}[htbp]
+    \centering
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_47_tuc_r.pdf}
+        \caption{47 Tuc}
+        \label{fig:img1}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_rubin_sv_38_7_r.pdf}
+        \caption{Rubin SV 38 7}
+        \label{fig:img2}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_fornax_dsph_r.pdf}
+        \caption{Fornax dSph}
+        \label{fig:img3}
+    \end{subfigure}
+    \vspace{1em}  % Vertical space between rows
+     \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_ecdfs_r.pdf}
+        \caption{ECDFS}
+        \label{fig:img4}
+    \end{subfigure}
+    \hfill
+    \vspace{1em}  
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_edfs_r.pdf}
+        \caption{EDFS}
+        \label{fig:img6}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_rubin_sv_95_-25_r.pdf}
+        \caption{Rubin SV 95 -25}
+        \label{fig:img7}
+    \end{subfigure}
+    \hfill
+    \begin{subfigure}[b]{0.3\textwidth}
+        \centering
+        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_seagull_r.pdf}
+        \caption{Seagull}
+        \label{fig:img8}
+    \end{subfigure}
+     \caption{Cumulative imaging depth expressed in terms of the $S/N=5$ limiting magnitude for unresolved sources for seven ComCam Deep Drilling Fields.}
+     \label{fig:dp1_fields_psf_maglim}
+\end{figure}
+
+The  processing and preparation of ComCam data for DP1 will take place during the first half of 2025, with an expected DP1 release data in June or July 2025. 
+The planned data products for DP1 are presented in table~ \ref{tab:data-preview-summary}. 
+Data products that are marked as a stretch goal, DRP ForcedSource Catalogs, Difference Images and DIA Catalogs,  will not be confirmed until close to the DP1 release date. 
+DP1 will not contain any DRP  SSP data products in order to focus on readiness for LSSTCam and on providing high quality live LSSTCam SSP Prompt data products as part of the ramp up of Alert Production starting roughly contemporaneously with the DP1 release.
+
+
+%%% rehash %%%
+ComCam observations that are expected to be included in Data Preview 1.
+
+Keep in mind that this is “expected” in a general sense.
+Exact image counts and coadd depths may change.
+
```

Comment:
Maybe no paragraph break is needed here.

Reply:
Sorry- this was scratch text that I forgot to remove -- corrected